spec/v3/spec.md

```diff
@@ -365,18 +365,16 @@ Additionally, this specification defines the following keys, which MAY be includ
 |`type`|The type of the tileset |a string with a value of either `overlay` or `baselayer`|
 |`version`|The version number of the tileset|a string containing a valid version according to [Semantic Versioning 2.0.0](https://semver.org/spec/v2.0.0.html) |
 
-The JSON object MAY also include any other keys with an arbitrary value. This specification recommends nesting all application-specific data in an object under a semi-unique key to avoid overlap with other application-specific data or keys that may be defined in future versions of this specification. For example, instead of including the custom fields `author` and `companyId` directly in the top level of the metadata object, they SHOULD be nested in another object under a key with your project or organization name.
-
```

Comment:
I just added this as an "insurance policy" to reduce the probability of breaking someone's code when adding a new property. So for example it could cause complications if we want to add `vector_layers` in a future version of this spec, but someone else already uses `vector_layers` with a different format.
So this is just a friendly reminder to not use common names, which is already best practice, but you know how lazy software engineers are 😛 

Prefixed values like you did in `planetiler` is also okay. It solves the same "problem". We can also change this paragraph to include something along the lines of _"Please prefix your keys in the root object"_. Would serve the exact same purpose of what I tried to do :)